platform/darwin/src/MGLStyle.h

```diff
@@ -194,8 +194,8 @@ MGL_EXPORT
 @property (nonatomic, strong) NS_SET_OF(__kindof MGLSource *) *sources;
 
 /**
- Values describing animated transitions to styling changes, either to the style URL 
- or to individual properties.
+ Values describing animated transitions to changes on a style's individual
+ paint properties.
```

Comment:
This sounds convoluted but more consistent with the individual transition properties’ documentation comments:

> The transition affecting any changes to the paint attributes of the style’s layers.

> This property corresponds to the [`transition`](https://www.mapbox.com/mapbox-gl-js/style-spec/#root-transition) property in the style JSON file format.